api/src/main/scala/Utils.scala

```diff
@@ -0,0 +1,111 @@
+package org.wikiwatershed.mmw.geoprocessing
+
+import spray.json._
+import spray.json.DefaultJsonProtocol._
+
+import geotrellis.proj4.{CRS, ConusAlbers, LatLng, WebMercator}
+
+import geotrellis.raster._
+import geotrellis.raster.rasterize._
+import geotrellis.vector._
+import geotrellis.vector.io._
+import geotrellis.spark._
+import geotrellis.spark.io._
+import geotrellis.spark.io.s3._
+
+import com.typesafe.config.ConfigFactory
+
+trait Utils {
+  val s3bucket = ConfigFactory.load().getString("geoprocessing.s3bucket")
+  val baseLayerReader = S3CollectionLayerReader(s3bucket, "")
+
+  /**
+    * Given a zoom level & area of interest, transform a list of raster
+    * filename into a [[TileLayerCollection[SpatialKey]]].
+    *
+    * @param   rasterIds  A list of raster filenames
+    * @param   zoom       The input zoom level
+    * @param   aoi        A MultiPolygon area of interest
+    * @return             [[TileLayerCollection[SpatialKey]]]
+    */
+  def createRasterLayerIds(rasterIds: List[String], zoom: Int, aoi: MultiPolygon) =
```

Comment:
Consider adding a return type to all non-private methods.

```scala
def createRasterLayerIds(
  rasterIds: List[String],
  zoom: Int, aoi: MultiPolygon
): Seq[TileLayerCollection[SpatialKey]] =
  rasterIds
    .map { str => LayerId(str, zoom) }
    .map { layer => fetchLayer(layer, aoi)}
```

Furthermore, this return types indicates that the method is converting Raster LayerIds to cropped Layers, not "creating Raster LayerIds". Consider renaming to something more accurate.